src/VirtoCommerce.SimpleExportImportModule.Web/module.manifest

```diff
@@ -5,6 +5,8 @@
     <version-tag></version-tag>
     <platformVersion>3.49.0</platformVersion>
     <dependencies>
+        <dependency id="VirtoCommerce.Catalog" version="3.35.0" />
```

Comment:
I think this dependency is redundant. Our module is dependent on pricing which depends on the catalog.

Reply:
I think instead we should add Catalog nuget excplicitly, because, for example, (I know it's almost impossible, but any way) if Pricing module will remove Catalog module from dependencies, then our will be broken. Compare it with Search module: we don't use it explicitly, so if Catalog module will replace it with another module, then our module will still working.